llvm/lib/Analysis/ValueTracking.cpp

```diff
@@ -815,8 +832,7 @@ void llvm::computeKnownBitsFromContext(const Value *V, KnownBits &Known,
     if (!isValidAssumeForContext(I, Q.CxtI, Q.DT))
       continue;
 
-    computeKnownBitsFromCmp(V, Cmp->getPredicate(), Cmp->getOperand(0),
-                            Cmp->getOperand(1), Known, Q);
+    computeKnownBitsFromICmpCond(V, Cmp, Known, Q, /*Invert=*/false);
```

Comment:
Thought the idea was move logic from `computeKnownBitsFromICmpCond` -> `computeKnownBitsFromCmp`, not use `computeKnownBitsFromICmpCond` for assumes.